google/cloud/spanner_dbapi/connection.py

```diff
@@ -37,11 +34,25 @@ class Connection:
     """
 
     def __init__(self, instance, database):
+        self._sessions_pool = BurstyPool()
+        self._sessions_pool.bind(database)
+
         self.instance = instance
         self.database = database
-        self.is_closed = False
 
         self._ddl_statements = []
+        self.transactions = []
+
+        self.is_closed = False
+        self.autocommit = True
+
+    def session(self):
+        """Get a Cloud Spanner session.
+
+        :rtype: :class:`google.cloud.spanner_v1.session.Session`
+        :returns: Cloud Spanner session object ready to use.
+        """
+        return self._sessions_pool.get()
```

Comment:
Method to get a session within `Cursor` object

Reply:
It would be better to name it `session_checkout` or so, to avoid future confusion with session factories.

Reply:
Done